core/src/main/java/com/amplifyframework/predictions/models/FaceLivenessSession.kt

```diff
@@ -34,7 +33,8 @@ class FaceLivenessSession(
         onChallengeResponseEvent(challengeResponseEvent)
     }
 
-    fun stopSession() {
-        stopLivenessSession.call()
+    @JvmOverloads
```

Comment:
Lets investigate why this is necessary.

Reply:
Seems like the consensus is that the UI library at a lower version is compiled expecting no argument.  If UI was compiled against this updated version of Amplify it would have a virtual reference to the correct function.  The JVM overloads are useful because Kotlin compiles down to the JVM.